AmplifyPlugins/API/Tests/AWSAPIPluginTests/Support/Decode/GraphQLResponseDecoder+DecodeDataTests.swift

```diff
@@ -10,7 +10,7 @@ import XCTest
 import AWSPluginsCore
 @testable import AmplifyTestCommon
 @testable import AWSAPIPlugin
-import AmplifyAsyncTesting
+import AmplifyAsyncTestingInternal
```

Comment:
I think all files in the integration level tests apps also have this change and build successfully, there should be some places that still call `import AmplifyAsyncTesting`